milli/src/search/mod.rs

```diff
@@ -315,14 +316,25 @@ impl<'a> SearchForFacetValues<'a> {
         search_query: Search<'a>,
         is_hybrid: bool,
     ) -> SearchForFacetValues<'a> {
-        SearchForFacetValues { query: None, facet, search_query, is_hybrid }
+        SearchForFacetValues {
+            query: None,
+            facet,
+            search_query,
+            max_values: MAX_NUMBER_OF_VALUES_PAR_FACET,
```

Comment:
```suggestion
            max_values: MAX_NUMBER_OF_VALUES_PER_FACET,
```

Reply:
I'm wondering if we shouldn't call this `DEFAULT_NUMBER_OF_VALUES_PER_FACET` given it is configurable now, but that's a nitpick